src/documents/en/faq.html.md

```diff
@@ -30,5 +28,11 @@ We also perform daily backups of your data, so we can restore them if needed.
 
 We provide 40GB of storage with each instance (well, technically 60GB are available, but we recommend for now not to use more than 40GB)
 
+## What can i do if I forget my password ?
+
+The "password recovery email", that you can activate in the login page of your cozy ("Forget password? Email me instructions"), will send you a link that will enable you to enter a new password. If needed, you will have to reenter this new password in the different apps of your cozy.
```

Comment:
Sorry but this is still wrong, this PR should'nt have been merged. You don't need to enter the new password of the server in the applications. You need to take a tour of all the applications that store a password to connect to a remote account (for example Emails, Konnectors, Kresus), and edit this account to enter again the password.